forms-flow-web/src/components/Bundle/steps/PreviewAndConfirm.js

```diff
@@ -118,7 +120,10 @@ const PreviewAndConfirm = React.memo(
                     rows={4}
                     variant="outlined"
                     className="text-field"
-                    value={""}
+                    value={ comment || ""}
```

Comment:
no need to add extra check here because already adding  ``` value || "" ```

Reply:
resolved